fileAnalyzer.h

```diff
@@ -3,14 +3,17 @@
 #include <iostream>
 #include <vector>
 #include <atomic>
+#include <ctime>
+#include <regex>
+#include <thread>
 
 namespace fs = std::filesystem;
 
 class fileAnalyzer
 {
 private:
     std::string dirName;
-    std::atomic<int> processedFilesCount = 0;
+    std::atomic<int> processedFilesCount  = 0;
     std::atomic<int> totalLines           = 0;
     std::atomic<int> blankLines           = 0;
     std::atomic<int> commentLines         = 0;
```

Comment:
Please use referense for input parameters for methods: 
void parse(const std::string filePath);
void changeDirName(const std::string newDir);